src/components/shared/thread/ActivityGraphFills.js

```diff
@@ -502,15 +502,13 @@ export class ActivityFillGraphQuerier {
   getSampleAndCpuRatioAtClick(
     cssX: CssPixels,
     cssY: CssPixels,
-    time: Milliseconds,
-    canvasBoundingRect: ClientRect
+    time: Milliseconds
   ): HoveredPixelState | null {
     const {
-      canvasPixelWidth,
       rangeFilteredThread: { samples, stackTable },
       greyCategoryIndex,
     } = this.renderedComponentSettings;
-    const devicePixelRatio = canvasPixelWidth / canvasBoundingRect.width;
+    const { devicePixelRatio } = window;
```

Comment:
I also changed some other locations like here to avoid calling `getBoundingClientRect`. This wasn't strictly necessary but this is a bit simpler.